pkg/ccl/backupccl/backup_test.go

```diff
@@ -435,9 +435,6 @@ func TestBackupRestoreSystemJobs(t *testing.T) {
 	conn := sqlDB.DB.(*gosql.DB)
 	defer cleanupFn()
 
-	// Get the number of existing jobs.
-	baseNumJobs := jobutils.GetSystemJobsCount(t, sqlDB)
```

Comment:
Does this still need to be removed?

Reply:
Yea -- that was needed to calculate the offset when the code was selecting all jobs. Now that it's just selecting `BACKUP`/`RESTORE` jobs, we don't need `baseNumJobs`